components/VotePanel/CastMultiVoteButtons.tsx

```diff
@@ -0,0 +1,198 @@
+import { Proposal, VoteKind } from '@solana/spl-governance'
+import { useState } from 'react'
+import Button, { SecondaryButton } from '../Button'
+import VoteCommentModal from '../VoteCommentModal'
+import {
+  useIsVoting,
+  useVoterTokenRecord,
+  useVotingPop,
+} from './hooks'
+import useRealm from '@hooks/useRealm'
+import { VotingClientType } from '@utils/uiTypes/VotePlugin'
+import useVotePluginsClientStore from 'stores/useVotePluginsClientStore'
+import useWalletOnePointOh from '@hooks/useWalletOnePointOh'
+import { useProposalVoteRecordQuery } from '@hooks/queries/voteRecord'
+import { useSubmitVote } from '@hooks/useSubmitVote'
+import { useSelectedRealmInfo } from '@hooks/selectedRealm/useSelectedRealmRegistryEntry'
+
+const useCanVote = () => {
```

Comment:
rather than copypasting this lets just export it from its own file and import it where it gets used. the file can just go in this folder.